aws-datastore/src/main/java/com/amplifyframework/datastore/network/SyncEngine.java

```diff
@@ -137,6 +140,11 @@ private void startModelSubscriptions() {
                         AmplifyException.TODO_RECOVERY_SUGGESTION
                     );
             }
+
+            // Notify Hub that we just updated the local storage.
+            HubEvent<? extends Model> receivedFromCloudEvent =
+                HubEvent.create(DataStoreChannelEventName.RECEIVED_FROM_CLOUD, mutation.model());
+            Amplify.Hub.publish(HubChannel.DATASTORE, receivedFromCloudEvent);
```

Comment:
One thought here is do we want to have this on a channel for DataStore or API? We've already seen customers asking about using the API category and DataStore together, so there might be merit to having everything going over the API channel for events from AppSync. OTOH we could always publish to both or switch it later. Food for thought.

Reply:
Since this event was initiated by the datastore, I thought to put it out onto the datastore channel.

I think there will be fewer hub events for the API category, overall. Although, there might be some good reasons to use Hub for subscription events, paging on list APIs, retries on recoverable error, things like this.